### PR TITLE
shield has less bomb armor, and no longer gives you immunity to bio, only half bio armor

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
shield appreantly had 100 bio armor, reduced to 50, so you get 50 bio armor with just a shield which is still pretty high
shield no longer gives you as much bomb protection to prevent stacking to become immune to grenades

## Why It's Good For The Game
grenade immunity bad
bio immunity very very bad

## Changelog
:cl:
balance: shield has less bomb armor, and no longer has bio armor immunity, only 50%.
/:cl:

### CODE REVIEW

**Comment by DerrickChamberlain** [comment]
Damn at this rate we may as well just remove the Shield seems useless now.

**Comment by David-Stormwell** [comment]
> Damn at this rate we may as well just remove the Shield seems useless now.

Dam I can't be immune to damages anymore...
Dam I'm going to have to cope with Marine nerfs because mother fucking Xeno players suck and Xeno is boring to play...

**Comment by DerrickChamberlain** [comment]
> > Damn at this rate we may as well just remove the Shield seems useless now.
> 
> Dam I can't be immune to damages anymore...
> Dam I'm going to have to cope with Marine nerfs because mother fucking Xeno players suck and Xeno is boring to play...
> 
> Though will say, a complete nerf from 100 bio is too much, same with total bomb armor being qdeled, keep some.

Just pointing out that there have been a few shield nerf PRs, will probably result in the death of the object so why even have it. IT . IS. A. SHIELD. Needs to actually do something defensive.

**Comment by AnCopper** [comment]
> > > Damn at this rate we may as well just remove the Shield seems useless now.
> > 
> > 
> > Dam I can't be immune to damages anymore...
> > Dam I'm going to have to cope with Marine nerfs because mother fucking Xeno players suck and Xeno is boring to play...
> > Though will say, a complete nerf from 100 bio is too much, same with total bomb armor being qdeled, keep some.
> 
> Just pointing out that there have been a few shield nerf PRs, will probably result in the death of the object so why even have it. IT . IS. A. SHIELD. Needs to actually do something defensive.

The thing is everyone uses the shield to play hyper-offensively.

File: code/game/objects/items/weapons/shields.dm [hunk 1]
```diff
@@ -137,7 +137,7 @@
 	flags_equip_slot = ITEM_SLOT_BACK
 	max_integrity = 400
 	integrity_failure = 100
-	soft_armor = list("melee" = 50, "bullet" = 50, "laser" = 0, "energy" = 100, "bomb" = 30, "bio" = 100, "rad" = 100, "fire" = 0, "acid" = 35)
+	soft_armor = list("melee" = 50, "bullet" = 50, "laser" = 0, "energy" = 100, "bomb" = 0, "bio" = 35, "rad" = 0, "fire" = 0, "acid" = 35)
```
Comment (Pariah919) [inline code/game/objects/items/weapons/shields.dm:140]:
```suggestion
	soft_armor = list("melee" = 50, "bullet" = 50, "laser" = 0, "energy" = 100, "bomb" = 15, "bio" = 50, "rad" = 0, "fire" = 0, "acid" = 35)
```

**Comment by Losenis** [comment]
Doesn't the 100 armor mean all damage is tranferred to the shield? Acid turrets MELT through shield hp somehow, so that might be the reason. I'd probably consider it a buff given I'd trade chunks of my health on burn damage to the shield ending up under the metal threshold or straight up breaking.

**Comment by InterroLouis** [comment]
Yes because it increases survivability. Which is its purpose.

**Comment by Al-1ce** [comment]
> Damn at this rate we may as well just remove the Shield seems useless now.

It's already insanely powerful as it is. You need either half the hive or one very determined Xeno to take down someone with a shield due to its inherent damage protection. At least once per round I see someone survive being swarmed by 4-6 Xenos purely due to the shield. A shield is meant to be protecting, but not solid vibranium.

**Comment by SplinterGP** [comment]
> 
> 
> > Damn at this rate we may as well just remove the Shield seems useless now.
> 
> It's already insanely powerful as it is. You need either half the hive or one very determined Xeno to take down someone with a shield due to its inherent damage protection. At least once per round I see someone survive being swarmed by 4-6 Xenos purely due to the shield. A shield is meant to be protecting, but not solid vibranium.

well it gives fucking 0.5 slowdown now

**Comment by Losenis** [comment]
A shield locks you out of pretty much any weapon that's better used 2handed. It better fucking be that useful for that gigantic tradeoff.

>At least once per round I see someone survive being swarmed by 4-6 Xenos purely due to the shield.

Keyword, survive. Not beat. He better run before that shield breaks or he's dead meat, because it takes one drone with larval to make him a dead man walking. I've even had illegal amounts of fun being defiler and just poking shield people with neurotox slash because they can't ward me off or chase due to the lack of a second hand to use any weapon + slowdown. Not only that, shield makes marines want to be at the front which inevitably force everyone to kill their DPS by getting aim mode forced on or risk the shield just tanking all the shots that'd have gone to xenos.

**Comment by Swagile** [comment]
shield:

- removes all 2hand usage (only p90, old lasgun, pistols, revolvers actually good due to this fact)
- added slowdown
- hard armor removed

its already nerfed as is and basically have to wear fast armor or else you can get warriored / shrikes / etc into a bad position

its exactly why i buffed old B18 to be medium speed instead of the highest movement debuff speed armor in the game, because armor means nothing if your as slow as a fucking turtle since low speed lets xenos get huge amount of free damage onto you and keep putting you in bad positions

ok cool you can juggle nades with shield but you have slowdown + no hard armor anymore so hilin nading, while effective, gets you killed if you make 1 bad play

and i don't even need to play anymore to know that

shit PR

**Comment by Losenis** [comment]
>revolver

Maybe if you're trying to Ocelot it because good luck trying to reload that shitshow with one hand

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**